src/configs/opa.rs

```diff
@@ -0,0 +1,34 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Deserialize, Serialize)]
+#[cfg_attr(
+    feature = "config-schema",
+    derive(schemars::JsonSchema),
+    schemars(deny_unknown_fields)
+)]
+#[serde(default)]
+pub struct OpaConfig<'a> {
+    pub format: &'a str,
+    pub version_format: &'a str,
+    pub symbol: &'a str,
+    pub style: &'a str,
+    pub disabled: bool,
+    pub detect_extensions: Vec<&'a str>,
+    pub detect_files: Vec<&'a str>,
+    pub detect_folders: Vec<&'a str>,
+}
+
+impl<'a> Default for OpaConfig<'a> {
+    fn default() -> Self {
+        OpaConfig {
+            format: "via [$symbol($version )]($style)",
+            version_format: "v${raw}",
+            symbol: "🪖 ",
```

Comment:
VS-Code and iTerm require two spaces this for proper spacing, or are there any other options? The emoji was only standardized recently:
```suggestion
            symbol: "🪖  ",
```

Reply:
I can't test on iTerm, but:
Two spaces is the correct amount in VSCode, but
two spaces is too much on kitty

Same thing happens with an alternative emoji, ⛑️ "Helmet with White Cross", even though that's existed for far longer.

What's the consensus on other modules that this happens to, one space or two spaces?

Reply:
Kitty is better at determining proper widths, but other terminals will not match it, so please go with two spaces.

Reply:
Updated and pushed.